terraform/environments/delius-iaps/ssm.tf

```diff
@@ -90,3 +90,16 @@ resource "aws_ssm_parameter" "im-interface-soap-odbc-pwd" {
   }
 }
 
+resource "aws_ssm_parameter" "ndelius-interface-ssm-param" {
```

Comment:
When it came to storing the parameters using a single resource, I was thinking more along the line of this
```
resource "aws_ssm_parameter" "ndelius_interface" {
  name = "/NDeliusInterface/Interface"
  type = "String"
  value = jsonencode({
    "user" : "dev-placeholder-ndelius-interface-user",
    "password" : "dev-placeholder-ndelius-interface-password",
    "replicapwdcoded" : "dev-placeholder-ndelius-interface-replicapasswordcoded",
    "soappwdcoded" : "dev-placeholder-ndelius-interface-soappasscoded",
    "pwdcoded" : "dev-placeholder-ndelius-interface-passwordcoded",
    "dsn" : "dev-placeholder-ndelius-interface-odbc-dsn",
    "uid" : "dev-placeholder-ndelius-interface-odbc-uid",
		...
  })
}
```

This would allow us to look up the parameter once and return all the things we need. then use something like `jq` to filter out the values in `terraform/environments/delius-iaps/templates/iaps-EC2LaunchV2.yaml.tftpl`